isis/src/lro/apps/lronacpho/LROCEmpirical.cpp

```diff
@@ -333,18 +342,26 @@ namespace Isis {
    *
    * @internal
    *   @history 2016-08-15 Victor Silva - Adapted from the lrowacpho application
-                              written by Kris Becker.
+   *            written by Kris Becker.
+   *
+   *   @history 2021-03-12 Victor Silva - Added b parameters for 2019 version of
+   *            LROC Empirical algorithm
+   *
    */
   LROCEmpirical::Parameters LROCEmpirical::extract( const DbProfile &profile) const {
     Parameters pars;
-    pars.a1 = toDouble(ConfKey(profile, "A1", toString(0.0)));
-    pars.a2 = toDouble(ConfKey(profile, "A2", toString(0.0)));
-    pars.a3 = toDouble(ConfKey(profile, "A3", toString(0.0)));
+
+    for (int i=0; i<4; i++)
+        pars.aTerms.push_back(toDouble(ConfKey(profile, "A" + toString(i), toString(0.0))));
+    for (int i=0; i<7; i++)
+        pars.bTerms.push_back(toDouble(ConfKey(profile, "B" + toString(i), toString(0.0))));
```

Comment:
I noticed Jesse suggested changing the `toString` call with the string literal `"0.0"`, and that you made the change, was this reverted somehow? Or do you need to explicitly cast to a `QString`?

Reply:
This must have gotten reverted when I corrected the merge issue. Thanks. Fixed in code.